code/src/pages/dashboard/dashboard-product/DashboardProductPage.types.ts

```diff
@@ -0,0 +1,3 @@
+export type DashboarProductContainerProps = {
+  productId: string;
+};
```

Comment:
already exist type with `productId: string`
I suggest to use it once everywhere as some `common` type and not to create it each time

Reply:
I think types for component props should be defined individually for each component rather than as `common` type. Defining them as `common` type could lead to problems when we need to change props for one of those components.

Reply:
> I think types for component props should be defined individually for each component rather than as `common` type. Defining them as `common` type could lead to problems when we need to change props for one of those components.

I partially agree with you